Library/Homebrew/os/linux/elf.rb

```diff
@@ -172,6 +201,14 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def patcher
+    @patcher ||= begin
+      Homebrew.install_bundler_gems!
+      require "patchelf"
+      PatchELF::Patcher.new to_s, logging: false
+    end
+  end
```

Comment:
```suggestion
  def patchelf_patcher
    @patchelf_patcher ||= begin
      Homebrew.install_bundler_gems!
      require "patchelf"
      PatchELF::Patcher.new to_s, logging: false
    end
  end
```

Reply:
i'll make these changes offline, run brew tests and push soon.